DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
                     ->end()
                 ->end()
+                ->arrayNode('content_listener')
+                    ->canBeDisabled()
```

Comment:
Instead, I think we should use a `booleanNode` and call it `use_content_listener`